azure-pipelines/pull-request-validation/common4j.yml

```diff
@@ -38,5 +38,5 @@ jobs:
     displayName: Run the unit tests.
     inputs:
       cwd: $(Build.SourcesDirectory)/common-java-root
-      tasks: common4j:test -Psugar=true
+      tasks: common4j:test -Psugar=true -PuseMockApiForNativeAuth=true
```

Comment:
Perhaps we can set the default value for this to true, so that we don't have to explicitly pass it. We can change the default to false once prod api integration testing is available.

Reply:
Boolean flags should all have a common default value of false as it allows a casual user of the library to use it without having to understand the intricacies of those flags. For this reason I have set the default value of this flag to false.